src/pages/moonsil/Login/Login.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import './Login.scss'; //sass랑 css파일은 이름을 붙여주지 않아도 된다.
+import { withRouter } from 'react-router-dom';
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: '',
+      pw: '',
+      idCondition: '',
+      pwCondition: '',
+    };
+  }
+
+  handleIdInput = e => {
+    this.setState({
+      id: e.target.value,
+      idCondition: this.state.id.indexOf('@') !== -1,
+    });
+  };
+
+  handlePwInput = e => {
+    this.setState({
+      pw: e.target.value,
+      pwCondition: this.state.pw.length >= 5,
+    });
+  };
```

Comment:
```suggestion
  handleInputs = e => {
    this.setState({
      [e.target.name]: e.target.value,
    });
  };
```
위와 같이 하나의 함수로도 사용할 수 있습니다. 
단, target이 되는 input 태그에 name을 추가해주셔야 합니다.!! 
id input에는 name="id" 이런식으로요!

유효성 부분은 더 효율적인 방법을 말씀드리겠습니다(다음 리뷰 참고)